client/globals.d.ts

```diff
@@ -24,7 +24,15 @@ declare const wcpaySettings: {
 		status?: string;
 		country?: string;
 		paymentsEnabled?: boolean;
-		deposits?: Array< any >;
+		deposits?: {
```

Comment:
With `Array< any >`, when I tried to look for the value of `wcpaySettings.accountStatus.deposits.completed_waiting_period`, TS will complain, so I thought I'll define `deposits` type more based on the response from WCPay server.

Come to think of it, this line could have been changed as simple as:
```
deposits?: Record< string, any >;
```

Which one is better you think?